libraries/AP_Math/AP_Math.cpp

```diff
@@ -395,6 +397,7 @@ float calc_lowpass_alpha_dt(float dt, float cutoff_freq)
 }
 
 #if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+// LCOV_EXCL_START
```

Comment:
Oh, people are really, really not going to like this.

I might put up with it - but I'm pretty sure others won't.

I really think we'll need to put up with less than 100% coverage.

Reply:
nha, I want 100%, so I will cheat ! ;-P
Ideally, I would like for AP_Math to have 100% coverage with unit_tests. But I guess it could wait a little or I can put another special #define in the test file to get those